test/camunda-cloud/no-loop.spec.js

```diff
@@ -71,25 +71,28 @@ const valid = [
     moddleElement: createModdle(createDefinitions(`
       <bpmn:process id="Process_1" isExecutable="true">
         <bpmn:startEvent id="StartEvent_1">
-          <bpmn:outgoing>SequenceFlow_1</bpmn:outgoing>
+          <bpmn:outgoing>SeqeuenceFlow_1</bpmn:outgoing>
```

Comment:
Feedback: I find it hard to assess the changes that we're testing here as I cannot _see_ the BPMN diaram.

Why don't we source the test samples from a diagram that I can open?

Please share the before and after diagrams that you used.

Reply:
Since there are so many test cases they each have their own diagram, we'd end up with hundreds of diagrams. I agree that it's difficult without being able to open the diagram. 😞

Reply:
# Before 🤡 

![before](https://github.com/camunda/bpmnlint-plugin-camunda-compat/assets/7633572/d772d840-a53e-40a3-9028-b24f87cab574)

# After

![after](https://github.com/camunda/bpmnlint-plugin-camunda-compat/assets/7633572/e1cdc7d3-829c-487e-abba-2a6b05c86610)

Reply:
> Since there are so many test cases they each have their own diagram, we'd end up with hundreds of diagrams. 

As long as we don't dynamically generate the contents, what's wrong with 100 test diagrams? :slightly_smiling_face:

Reply:
Looking at _before_ and _after_ what did we fix? And why don't we keep before as a valid test case, and add another one (after)? What does adding the `X` gateway brings to the table?

A general good practice could be to give each of these tests human readable names:

> NOT:
> * call activity
>
> BUT:
> * call activity (calling other process)
> * call activity (calling dynamically computed process)
> * ...
> * call activity (calling itself)

Reply:
I do understand that what we're testing here is 

> call activity (calling other process)

Reply:
We could have a separate folder for each of the rules.